src/Components/CriticalCareRecording/PressureSore/styles.css

```diff
@@ -0,0 +1,3 @@
+input[type="checkbox"]:checked ~ .dot {
+  transform: translateX(100%);
+}
```

Comment:
Shoud this be inside `CriticalCareRecording/Recording/styles.css` ? or both styles merged for critical care task page

Reply:
I don't know how to do this in Tailwind CSS. So if you can go ahead.

Reply:
Ohk let me try this. 😅